travis.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$JOB_ENV" == "BuildAndTests" ]; then
+  echo "### Compiling all targets."
+  mkdir -p build;
+  cd build;
+  cmake ..
+  make
+
+  echo "### Running test."
+  ./testVirtualJukebox
+
+  echo "### Running main executable."
+  ./VirtualJukebox
```

Comment:
I dont think we want to execute the main app? eventually this will create a webserver etc.

Reply:
true.. removing it